build/BuildSteps.cs

```diff
@@ -189,6 +189,11 @@ public static void BuildExtensionsBundle(BuildConfiguration buildConfig)
                 FileUtility.EnsureDirectoryExists(Directory.GetParent(buildConfig.PublishBinDirectoryPath).FullName);
                 Directory.Move(Path.Combine(buildConfig.PublishDirectoryPath, "bin"), buildConfig.PublishBinDirectoryPath);
             }
+
+            // Temporary fix to copy assembly needed for cosmosDb extension

```

Comment:
SQL extension?

And this isn't really a temporary fix - unless SqlClient gets rid of the requirements for these then it'll be a permanent requirement for our extension.

Reply:
Cosmos needs it as well :). Also i think the fix is temporary because the ideal change should go in host or the sdk package.